index.js

```diff
@@ -58,13 +59,16 @@ var templates = get_templates(p);
  */
 var mailCredentials = {
   port: process.env.WATCHMEN_AUTH_NODEMAILER_PORT,
-  host: process.env.WATCHMEN_AUTH_NODEMAILER_HOST,
-  auth: {
-    user: process.env.WATCHMEN_AUTH_NODEMAILER_USER,
-    pass: process.env.WATCHMEN_AUTH_NODEMAILER_PASS
-  }
+  host: process.env.WATCHMEN_AUTH_NODEMAILER_HOST
 };
 
+if (process.env.WATCHMENT_AUTH_NODEMAILER_NO_AUTH != 'true') {
```

Comment:
that won't work because 'false' would evaluate to boolean true. Maybe just ..._NO_AUTH == 'false' would make it simpler I'd say. What do you think?